setu/deeplink.py

```diff
@@ -62,12 +63,16 @@ def create_payment_link(
 
         # Generate required headers
         headers = generate_setu_headers(
```

Comment:
Can we move `headers` to an instance variable and reset it with a token fetch call by catching 403 Invalid Token scenarios? Otherwise, user would have to handle this manually and re-initialize the object entirely.

Reply:
Moved to an instance variable. But as of now not supporting the 403 handling.